README.md

```diff
@@ -15,6 +15,9 @@ GOV.UK Pay Frontend application (Node.js)
 | `WORLDPAY_3DS_FLEX_DDC_LIVE_URL`       |    X     |                                                               | URL for Device Data Collection (DDC) initiation in LIVE                                                                                               |
 | `WORLDPAY_3DS_FLEX_CHALLENGE_TEST_URL` |    X     | `https://secure-test.worldpay.com/shopper/3ds/challenge.html` | Pointing to Worldpay's TEST 3ds flex challenge URL.                                                                                                   |
 | `WORLDPAY_3DS_FLEX_CHALLENGE_LIVE_URL` |    X     |  `https://centinelapi.cardinalcommerce.com/V2/Cruise/StepUp`  | Pointing to Worldpay's LIVE 3ds flex challenge URL.                                                                                                   |
+| `CSP_USE_CARD_PAYMENT_RULES`           |          | false/undefined                                               | Apply card payment contest security policy headers.                                                                                                   |
```

Comment:
Specifically, this controls whether a CSP header (whether `Content-Security-Policy` or `Content-Security-Policy-Report-Only`), right? I think the name could be tweaked to make this clearer.